references/cli/env.go

```diff
@@ -202,26 +222,23 @@ func SetEnv(args []string, ioStreams cmdutil.IOStreams) error {
 	return nil
 }
 
-// GetEnv gets environment by name or current environment
+// GetFlagEnvOrCurrent gets environment by name or current environment
 // if no env exists, then init default environment
-func GetEnv(cmd *cobra.Command) (*types.EnvMeta, error) {
+func GetFlagEnvOrCurrent(cmd *cobra.Command, args common.Args) (*types.EnvMeta, error) {
```

Comment:
I prefer the original function name as I am confused about `Flag` and `Current` in the name.

Reply:
I think developer can get more infomation in func name, or the func argument `*cobra.Command` are confusing to caller.